proto/api.proto

```diff
@@ -234,6 +234,7 @@ message EndTransactionRequest {
   // public-facing KV API.
   optional SplitTrigger split_trigger = 3;
   optional MergeTrigger merge_trigger = 4;
+  optional ChangeReplicasTrigger change_replicas_trigger = 5;
```

Comment:
You're going to need to update kv/db.go and kv/db_test.go to properly ignore this value if set through the public-facing API. We probably should make the trigger a proto buf message itself and move its definition to internal.proto so we don't have to keep adding these cases. Might as well add a TODO about that.